### ISSUE TITLE
ctrl+c raises an exception console

### ISSUE THREAD

**Issue by jvenezia** [Bastes/hrk#14]
When I use hrk to run a console, ctrl+c raises an exception, instead of letting heroku clean the current command.

**With hrk:**
```
hrk run console -a production
> ctrl+c
irb(main):001:0> .../gems/hrk-1.0.3/lib/hrk/heroku.rb:10:in `system': Interrupt
```

**Without hrk:**
```
heroku run console -a production
> ctrl+c
>
```

**Comment by Bastes** [comment]
Good news everyone: that one's fixed in 1.0.5 ^^

**Comment by jvenezia** [comment]
Awesome ! TY
